packages/wrangler/src/index.tsx

```diff
@@ -347,6 +351,39 @@ export async function main(argv: string[]): Promise<void> {
                 "utf-8"
               )
             );
+            if (
+              !packageJsonContent.scripts?.start &&
+              !packageJsonContent.scripts?.deploy &&
+              shouldCreatePackageJson
+            ) {
+              await writeFile(
+                pathToPackageJson,
+                JSON.stringify(
+                  {
+                    ...packageJsonContent,
+                    scripts: {
+                      ...packageJsonContent.scripts,
+                      start: "wrangler dev src/index.ts",
+                      deploy: "wrangler publish src/index.ts",
+                    },
+                  },
+                  null,
+                  "  "
```

Comment:
NIT: we can just use `2` here, which is easier to see than having to count the number of spaces in the string.

Reply:
I agree with this, but previous implementations were using `"  "` so if we change this one, I think the rest should be consistent as well.